src/index.css

```diff
@@ -52,3 +52,22 @@ body{
     /* height: 56vh; */
   }
 }
+/* Background of the scrollbar track */
```

Comment:
Since we're using TailwindCSS, You can simply use `@apply` directive to add custom styles.

Eg: 
```css
::-webkit-scrollbar-track {
  @apply bg-gray-200;
}

::-webkit-scrollbar-thumb {
  @apply bg-black hover:bg-black;
}

* {
  @apply select-none;
}
```